### ISSUE TITLE
Crash with firebase-core

### ISSUE THREAD

**Issue by qyxghcl** [firebase/quickstart-android#1002]
When I add firebase-core (com.google.firebase:firebase-core:16.0.4) to my App, the phone which is equal or greater than android 9.0 version  and without GooglePlay makes a crash even if I wrote nothing. What can I do now? I  can not decide peoples device and let them to fix Google Service.

**Comment by samtstern** [comment]
@qyxghcl can you tell me a little more so that we can figure out what the problem is?  What is the stack trace you get from the crash?  Does this happen if you run your app on an emulator without Google Play services or only on a real device?

**Comment by qyxghcl** [comment]
Yes, I run it on a real device. First I found this happened on a Xiaomi Mi 8(MIUI 10.1.2.0, Android 9) .Then    I tried it on other devices which were equals or higher than Android 9 without Google Service , the performance was the same.  I  think the stack trace "too many binders sent to SYSTEM" may result it. But  I can do nothing.

**Comment by qyxghcl** [comment]
Yes, I run it on a real device. First I found this happened on a Xiaomi Mi 8(MIUI 10.1.2.0, Android 9) .Then    I tried it on other devices which were equals or higher than Android 9 without Google Service , the performance was the same.  I  think the stack trace "too many binders sent to SYSTEM" may result it. But  I can do nothing.

**Comment by qyxghcl** [comment]
Yes, I run it on a real device. First I found this happened on a Xiaomi Mi 8(MIUI 10.1.2.0, Android 9) .Then    I tried it on other devices which were equals or higher than Android 9 without Google Service , the performance was the same.  I  think the stack trace "too many binders sent to SYSTEM" may result it. But  I can do nothing.

**Comment by qyxghcl** [comment]
Yes, I run it on a real device. First I found this happened on a Xiaomi Mi 8(MIUI 10.1.2.0, Android 9) .Then    I tried it on other devices which were equals or higher than Android 9 without Google Service , the performance was the same.  I  think the stack trace "too many binders sent to SYSTEM" may result it. But  I can do nothing.

**Comment by qyxghcl** [comment]
Yes, I run it on a real device. First I found this happened on a Xiaomi Mi 8(MIUI 10.1.2.0, Android 9) .Then    I tried it on other devices which were equals or higher than Android 9 without Google Service , the performance was the same.  I  think the stack trace "too many binders sent to SYSTEM" may result it. But  I can do nothing.

**Comment by qyxghcl** [comment]
Yes, I run it on a real device. First I found this happened on a Xiaomi Mi 8(MIUI 10.1.2.0, Android 9) .Then    I tried it on other devices which were equals or higher than Android 9 without Google Service , the performance was the same.  I  think the stack trace "too many binders sent to SYSTEM" may result it. But  I can do nothing.

**Comment by qyxghcl** [comment]
Yes, I run it on a real device. First I found this happened on a Xiaomi Mi 8(MIUI 10.1.2.0, Android 9) .Then    I tried it on other devices which were equals or higher than Android 9 without Google Service , the performance was the same.  I  think the stack trace "too many binders sent to SYSTEM" may result it. But  I can do nothing.

**Comment by samtstern** [comment]
@qyxghcl can you please provide the full `logcat` output or a bug report from a crash like this?  Also I am going to move this issue over to the Firebase Android SDK repo.
